scipy/stats/tests/test_distributions.py

```diff
@@ -125,6 +125,25 @@ def test_vonmises_numerical():
     assert_almost_equal(vm.cdf(0), 0.5)
 
 
+def test_support():
+    """gh-6235"""
+    def check_open_support(rvs, args):
+        dist = getattr(stats, rvs)
+        args = (.5,)*dist.numargs
```

Comment:
This redefines the `args` parameter. Note that you can use predefined arguments which are known to make sense for all distributions from https://github.com/scipy/scipy/blob/master/scipy/stats/_distr_params.py.